R/importWQP.R

```diff
@@ -57,8 +57,9 @@ importWQP <- function(obs_url, zip=TRUE, tz=""){
   if(!file.exists(obs_url)){
     
     if(zip){
+      message("zip encoding access still in development")
       temp <- tempfile()
-      # options(timeout = 120)
+      options(timeout = 120)
```

Comment:
You have led me down a rabbit hole where I'm trying to figure out the difference between R option's timeout and curlSetOpt(timeout = 200)

```
curlSetOpt(timeout = 120)
#vs:
options(timeout = 120)
```
@jiwalker-usgs any insight?